bonita/content.md

```diff
@@ -14,397 +14,281 @@ Bonita (called Bonita BPM till 7.5) is an open-source business process managemen
 
--	Launch the new container pointing towards the copy of DB and filesystem
+`DB_NAME` default value is bonitadb.
 
-	-	If < 7.3.0
+`DB_USER` default value is bonitauser.
 
-	```console
-	$ docker run --name=bonita_postgres --link mydbpostgres:postgres -e "DB_NAME=newbonitadb" -e "DB_USER=newbonitauser" -e "DB_PASS=newbonitapass" -v "$PWD"/bonita_migration:/opt/bonita/ -d -p 8081:8080 %%IMAGE%%:7.2.4
-	```
+`DB_PASS` default value is bonitapass.
 
-	-	If >= 7.3.0
+### BIZ_DB_NAME, BIZ_DB_USER, BIZ_DB_PASS
 
-	```console
-	$ docker run --name=bonita_postgres --link mydbpostgres:postgres -e "DB_NAME=newbonitadb" -e "DB_USER=newbonitauser" -e "DB_PASS=newbonitapass" -d -p 8081:8080 %%IMAGE%%:2021.2-u0
-	```
+These variables are used in conjunction to create a new user, set that user's password and create the bonita business database.
 
--	Reapply specific configuration if needed, for example with a version >= 7.3.0 :
+`BIZ_DB_NAME` default value is businessdb.
 
-	```console
-	$ docker exec -ti bonita_postgres /bin/bash
-	```
+`BIZ_DB_USER` default value is businessuser.
 
-	```console
-	$ cd /opt/bonita/BonitaCommunity-2021.2-u0/setup
-	$ ./setup.sh pull
-	$ TENANT_LOGIN=tech_user
-	$ TENANT_PASSWORD=secret
-	$ PLATFORM_LOGIN=pfadmin
-	$ PLATFORM_PASSWORD=pfsecret
-	$ sed -e 's/^#userName\s*=.*/'"userName=${TENANT_LOGIN}"'/' \
-	      -e 's/^#userPassword\s*=.*/'"userPassword=${TENANT_PASSWORD}"'/' \
-	      -i platform_conf/current/tenants/1/tenant_engine/bonita-tenant-community-custom.properties
-	$ sed -e 's/^platform.tenant.default.username\s*=.*/'"platform.tenant.default.username=${TENANT_LOGIN}"'/' \
-	      -e 's/^platform.tenant.default.password\s*=.*/'"platform.tenant.default.password=${TENANT_PASSWORD}"'/' \
-	      -i platform_conf/current/platform_portal/platform-tenant-config.properties
-	$ sed -e 's/^#platformAdminUsername\s*=.*/'"platformAdminUsername=${PLATFORM_LOGIN}"'/' \
-	      -e 's/^#platformAdminPassword\s*=.*/'"platformAdminPassword=${PLATFORM_PASSWORD}"'/' \
-	      -i platform_conf/current/platform_engine/bonita-platform-community-custom.properties
-	$ sed -i -e 's/^#GET|/GET|/' -e 's/^#POST|/POST|/' -e 's/^#PUT|/PUT|/' -e 's/^#DELETE|/DELETE|/' -i platform_conf/current/tenants/1/tenant_portal/dynamic-permissions-checks.properties
-	$ ./setup.sh push
-	```
+`BIZ_DB_PASS` default value is businesspass.
 
-	```console
-	$ docker restart bonita_postgres
-	```
 
--	Specific consideration regarding migration to Java 11 in Bonita 7.9
+## Logger configuration
 
-	Bonita 7.9 docker image runs with Java 11. If you are migrating from an earlier version which runs Java 8, you should validate on a test environment that your custom code is compatible. Aside from just code incompatibility, special attention has to be given to the dependencies of the custom code, as they might not work in Java 11.
+The logger can be configured by mounting a volume on folder `/opt/bonita/conf/logs` containing the configuration files.
```

Comment:
```suggestion
**Since 2022.1**

The logger can be configured by mounting a volume on folder `/opt/bonita/conf/logs` containing the configuration files.
```